### PR TITLE
Adapt 1938  horizontal image options

### PR DESCRIPTION
**Pull request by katrialesser**
# READY FOR REVIEW

# Summary
- Add horizontal card image styles and make the class available

# Review By (Date)
- 3/8/2021
# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- low, no actual visible change, foundational work for later

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Clear caches
3. Edit a card, and see the field with 3 options: small square, small circle, and large rectangle
4. verify that the wrapper classes are showing on the grandparent element of the su-card.
5. No visible/scss changes have been made yet.

## Front End Validation
- No front end validation done at this point, this is back-end work to prepare for front-end work later

### Code
- [x] Are the naming conventions following our standards?
- [x] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? No

# Associated Issues and/or People
- JIRA ticket(s) ADAPT-1938
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: saa_victoria_subtheme.theme [hunk 1]
```diff
@@ -35,6 +35,9 @@ function saa_victoria_subtheme_preprocess_ds_entity_view(&$variables) {
     // get card variant info - if null, will use the 3rd value
     $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'card_type', 'su-card-wrapper--icon');
     $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'card_border_style', 'su-card-wrapper--border-regular');
+    $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'horizontal_card_image_style');
+    dpm($variables);
```
Comment (jenbreese) [inline saa_victoria_subtheme.theme]: The dpm shouldn't be here. :)

**Comment by jenbreese** [comment]
I thought the class names should be: su-card-wrapper-horizontal--small-circle.  The -- for the last option? 
Edit:
su-card-wrapper-horizontal--small-square
su-card-wrapper-horizontal--large-rectangle

I tried all three and the class changes as I'd expect. Re-editing the page does not clear the options.

File: saa_victoria_subtheme.theme (hunk 1 above)
Comment (katrialesser) [reply saa_victoria_subtheme.theme]: oh woops! yes that wasn't supposed to be there haha.

**Review by jenbreese — APPROVED** [review]
GTG

File: saa_victoria_subtheme.theme (hunk 1 above)
Comment (jbickar) [reply saa_victoria_subtheme.theme]: Let he or she who has never left a stray `dpm()` cast the first stone...

Comment (jenbreese) [reply saa_victoria_subtheme.theme]: No stones thrown. :) I've done the old `console.log()` before.

**Pull request merged**